src/ioctls/vcpu.rs

```diff
@@ -2826,4 +2882,199 @@ mod tests {
         }
         assert!(vcpu.vcpu_init(&kvi).is_ok());
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_coalesced_pio() {
+        use crate::IoEventAddress;
+        use std::io::Write;
+
+        const PORT: u64 = 0x2c;
+        const DATA: u64 = 0x39;
+        const SIZE: u32 = 1;
+
+        #[rustfmt::skip]
+        let code = [
+            0xe6, 0x2c,   // out 0x2c, al
+            0xf4,         // hlt
+            0xe6, 0x2c,   // out 0x2c, al
+            0xf4,         // hlt
+        ];
+
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        if !vm.check_extension(Cap::CoalescedPio) {
+            return;
```

Comment:
Can we replace this `return` with a `panic`, to make sure we're actually running the test in CI?